src/Settings.sol

```diff
@@ -22,13 +32,50 @@ contract Settings is ISettings, Owned {
     /// @param _owner: address of the owner of the contract
     constructor(address _owner) Owned(_owner) {}
 
+    /*//////////////////////////////////////////////////////////////
+                                 VIEWS
+    //////////////////////////////////////////////////////////////*/
+
+    /// @inheritdoc ISettings
+    function isWhitelistedTokens(address _token)
```

Comment:
**Suggestion:** Could be named `isTokenWhitelisted`

Reply:
Agree would be better